app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PagesController < ApplicationController
+  def home; end
+end
```

Comment:
- [ ] [Optional] [Highly recommended] I can see that you used this controller and make a view and route for it to make it the main page (root route), It would be much better if you used the user index page as your main page instead.